locales/en/email-strings.ftl

```diff
@@ -90,3 +90,50 @@ email-breach-alert-all-data-points-title = Your exposed data:
 email-breach-alert-all-next-steps-lead = We’ll guide you step-by-step on how to resolve this data breach.
 email-breach-alert-all-next-steps-cta-label = Let’s get started
 email-breach-alert-all-next-steps-button-dashboard = Go to Dashboard
+
```

Comment:
Logging this from standup: apparently this email is only going to US users for now, and thus doesn't need localisation yet... (i.e. it can go to `locales-pending`.)

Reply:
I suppose so, yeah.